.github/workflows/test.yaml

```diff
@@ -6,7 +6,7 @@ jobs:
     strategy:
       matrix:
         go-version: [1.19.x]
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest]
```

Comment:
dropped mac and window because I don't think we ship the manager running for those architectures, happy to revert if necessary